### PR TITLE
add pre-rendering of svelte templates to run-bootstrap.sh

### PR DESCRIPTION
**Pull request by escattone**

### CODE REVIEW

File: bin/run-bootstrap.sh [hunk 1]
```diff
@@ -4,5 +4,7 @@ set -ex
 
 # install Node dependencies
 npm run development && npm run webpack:build
+# pre-render the svelte-based templates
+python manage.py compilejsi18n && npm run webpack:build:pre-render && python manage.py collectstatic --noinput
```
Comment (akatsoulas) [inline bin/run-bootstrap.sh:8]: Why do we need `compliejsi18n` and `collectstatic` here?

Comment (akatsoulas) [reply bin/run-bootstrap.sh:8]: even if we do need them, let's add the compilejsi18n first, then group together all the npm commands and leave the collectstatic to run after npm

**Review by akatsoulas — APPROVED** [review]
r+wc

File: bin/run-bootstrap.sh (hunk 1 above)
Comment (escattone) [reply bin/run-bootstrap.sh:8]: @akatsoulas Without running `compilejsi18n` prior, the `npm run webpack:build:pre-render` will fail, but perhaps we can remove the `collectstatic` step? I'll look into that.

**Pull request merged**